packages/ui-components/package.json

```
@@ -0,0 +1,108 @@
+{
+  "name": "ui-components",
+  "version": "0.0.1",
+  "description": "Reusable UI Components for Agoric Dapps, built with React and MaterialUI",
+  "main": "dist/index.js",
```

This is probably the source of the import problems.  React components can't easily be imported from bundles, IIUC.  The build step has to be performed only once, in the app that imports all the other components.  Otherwise, your app can't tree-shake the components that it doesn't use that are already bundled with other things.

If this change doesn't fix it for you, you might have instead to import the components as `import NatAmountInput from '@agoric/ui-components/src/components/NatAmountInput'`

```suggestion
  "main": "src/index.js",
```

---

I tried using the `src` directly, and it doesn't work in dapp-token-economy. I get an error message telling me to change my Babel settings to allow for JSX, but since we haven't ejected, I haven't found an easy way to change the Babel settings :/ There must be a way to bundle it correctly, though, since we can import MaterialUI components from packages.